cupy/_core/_cub_reduction.pyx

```diff
@@ -28,9 +28,13 @@ cdef function.Function _create_cub_reduction_function(
         _kernel._TypeMap type_map, preamble, options):
     # A (incomplete) list of internal variables:
     # _J            : the index of an element in the array
-
-    # static_assert needs at least C++11 in NVRTC
-    options += ('--std=c++11',)
+    
```

Comment:
```suggestion

```

Reply:
@takagi Seems like cuda-build pre-review may have some issues? See:https://github.com/cupy/cupy/actions/runs/4220966825/jobs/7327741747#step:7:119

Reply:
yeah, it's going to have a patch for now at #7417.